hydradx/model/amm/stableswap_amm.py

```diff
@@ -163,6 +163,22 @@ def price_at_balance(self, balances: list, d: float, i: int = 1, j: int = 0):
 
         return p
 
+    def share_price(self, numeraire: int = 0):
```

Comment:
I would prefer numeraire to be a string, just because it's consistent with the rest of the codebase. Is there an advantage to using an integer?

Reply:
changed to string